samples/glean/src/main/java/org/mozilla/samples/glean/MainActivity.kt

```diff
@@ -53,16 +62,27 @@ open class MainActivity : AppCompatActivity() {
 
         // Handle logic for the "test-color" experiment on click.
         buttonCheckExperiments.setOnClickListener {
-            textViewExperimentStatus.setBackgroundColor(Color.WHITE)
-            textViewExperimentStatus.text = getString(R.string.experiment_not_active)
+            onExperimentsUpdated()
+        }
+    }
+
+    /**
+     * This function will be called by the ExperimentUpdateListener interface when the experiments
+     * are updated.
+     */
+    override fun onExperimentsUpdated() {
```

Comment:
Can you add a comment stating that this is not relevant to the Glean SDK but to the experiments library?